Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- debootstrap-1.0.125.orig/debootstrap
+++ debootstrap-1.0.125/debootstrap
@@ -48,8 +48,8 @@ PRIVATEKEY=""
 CACHE_DIR=""
 INRELEASE_PATH=""
 
-DEF_MIRROR="http://deb.debian.org/debian"
-DEF_HTTPS_MIRROR="https://deb.debian.org/debian"
+DEF_MIRROR="http://deb.kaisenlinux.org"
+DEF_HTTPS_MIRROR="https://deb.kaisenlinux.org"
 
 # set $CONTAINER
 detect_container
--- /dev/null
+++ debootstrap-1.0.125/scripts/kaisen
@@ -0,0 +1,17 @@
+mirror_style release
+download_style apt
+finddebs_style from-indices
+variants - buildd fakechroot minbase
+keyring /usr/share/keyrings/kaisen-archive-keyring.gpg
+default_mirror https://deb.kaisenlinux.org
+
+# include common settings
+if [ -e "$DEBOOTSTRAP_DIR/scripts/debian-common" ]; then
+ . "$DEBOOTSTRAP_DIR/scripts/debian-common"
+elif [ -e /debootstrap/debian-common ]; then
+ . /debootstrap/debian-common
+elif [ -e "$DEBOOTSTRAP_DIR/debian-common" ]; then
+ . "$DEBOOTSTRAP_DIR/debian-common"
+else
+ error 1 NOCOMMON "File not found: debian-common"
+fi
