Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- debootstrap-1.0.137.orig/scripts/debian-common
+++ debootstrap-1.0.137/scripts/debian-common
@@ -113,6 +113,11 @@ first_stage_install () {
 }
 
 second_stage_install () {
+	#Ensure of /lib64 symlink has been created
+	if [ ! -L "$TARGET/lib64" ]; then
+		ln -sfr "$TARGET/usr/lib64" "$TARGET/lib64"
+	fi
+
 	in_target /bin/true
 
 	setup_dynamic_devices
